packages/components/src/range-control/styles/range-control-styles.ts

```diff
@@ -290,13 +291,31 @@ export const Tooltip = styled.span< TooltipProps >`
 	) }
 `;
 
+const inputNumberWidth = ( { size }: InputNumberProps ) => {
+	const sizes = {
+		default: space( 16 ),
+		'__unstable-large': space( 20 ),
+	};
+
+	if ( size === '__unstable-large' ) {
+		return css( {
+			width: `${ sizes[ '__unstable-large' ] } !important`,
+		} );
+	}
+
+	return css`
+		width: ${ sizes.default } !important;
+	`;
+};
```

Comment:
Are these `!important`s necessary?

Reply:
These were put into place[ to fix a visual regression a couple of years ago](https://github.com/WordPress/gutenberg/pull/28816). While we haven't yet completed the long term fix recommended at the time of refactoring `RangeControl` to use `HStack`, it does look like other more recent changes have alleviated the original issue. Without these `!importants` I'm not able to replicate the original problem shows in that PR, so I've removed them for now.